solpolpy/core.py

```diff
@@ -36,6 +38,9 @@ def resolve(input_data: t.Union[t.List[str], NDCollection], out_system: str) ->
         - "fourpol": For observations taken at sequence of four polarizer angles, i.e. 0°, 45°, 90° and 135°.
         - "npol": Set of images taken at than three polarizing angles other than MZP
 
+    imax_effect : Boolean
```

Comment:
We should add something in the docs about what the imax effect is. You might link to something in the docs webpages from here for a full explanation.

Reply:
We can refer our paper once it's out. Till then we have some poster (PUNCH 4) and abstract (AGU) to refer to.